tests/Backend/SOX/MergeRequestsTest.php

```diff
@@ -682,6 +682,106 @@ public function testMrWithConflictCantBeMergedButAfterResetCan(): void
         $this->assertBranchIsDeleted($branchId);
     }
 
+    public function testDeletedRowInBranchIsDeletedInDefaultAfterMerge(): void
```

Comment:
@romanbracinik frci ti to ? me to pada na `Row firstRow already exists`. nema ten `rowId` byt nejak unique?

Reply:
Row ma unique na `idProjec,IdBranch,IdComponent,configurationId,rowId`

<img width="2083" alt="Screenshot 2024-05-13 at 15 56 55" src="https://github.com/keboola/storage-api-php-client/assets/6448364/de8cc548-adc8-4173-83f7-3919ae8d4b6f">

Teraz som to znova spustil a preslo to

Reply:
tak su asi levej. jdu si to debugnout